public/locales/en/common.json

```diff
@@ -345,7 +345,23 @@
       "notMatchingAllFilters": "Showing farms matching at least one filter",
       "protocol": "Protocol",
       "stakedToken": "{{ token }} staked",
-      "token": "Token"
+      "token": "Token",
+      "details": "Details",
+      "tokensDeposited": "You have {{amount}} pTokens",
+      "pToken": "pTokens represent your appreciating share of the tokens in the Jar",
+      "metamaskAdd": "Add to Metamask",
+      "jarAddress": "Jar Address",
+      "farmAddress": "Farm Address",
+      "ratio": "Ratio",
+      "pending": "Pending",
+      "compounding": "Compounding",
+      "baseApr": "Base APR",
+      "ratioTooltip": "The jar's ratio is the amount of underlying deposit tokens per pToken",
+      "pendingTooltip": "This is the USD amount that your deposit is expected to grow upon the next re-investment of rewards",
+      "compoundingTooltip": "This is the most approximate frequency at which this jar performs re-investments",
+      "baseAprTooltip": "This is the base APR of the jar, without the effects of compounding",
+      "apyBreakdown": "APY breakdown",
+      "yieldRates": "Yield rates"
```

Comment:
@larrythecucumber321 a little request to keep these sorted alphabetically – easier to parse. VS Code can do that for you (highlight lines, sort).